corehq/apps/reports/standard/cases/basic.py

```diff
@@ -66,7 +66,8 @@ def _build_query(self):
         if self.case_status:
             query = query.is_closed(self.case_status == 'closed')
 
-        if self.request.can_access_all_locations and EMWF.show_all_data(mobile_user_and_group_slugs):
+        if self.request.can_access_all_locations and (EMWF.show_all_data(mobile_user_and_group_slugs) or
+                                                      not any(mobile_user_and_group_slugs)):
```

Comment:
I'd use `EMWF.all_users_selected` here too.

Aside: Now that I'm seeing it in action, I wonder if there's a better name, maybe `dont_filter_users` or `no_users_selected` or something.  `all_users_selected` makes it sound like the user actively selected "All Users".

Reply:
https://github.com/dimagi/commcare-hq/pull/21923/commits/15012b15856f3d04245f062b02e839cb4ad243bd

Reply:
I did 'no_filters_selected', does that sound good? https://github.com/dimagi/commcare-hq/pull/21923/commits/ee86619c3e9766650358868205af413e50e7c253